python_modules/dagster/dagster/_core/execution/plan/external_step.py

```diff
@@ -80,13 +80,13 @@ def launch_step(
         file_manager = LocalFileManager(".")
         events_file_handle = LocalFileHandle(events_file_path)
         events_data = file_manager.read_data(events_file_handle)
-        all_events = deserialize_value(pickle.loads(events_data))
+        all_events = cast(Sequence[EventLogEntry], deserialize_value(pickle.loads(events_data)))
 
         for event in all_events:
             # write each pickled event from the external instance to the local instance
             step_context.instance.handle_new_event(event)
             if event.is_dagster_event:
-                yield event.dagster_event
+                yield event.dagster_event  # type: ignore  # (possible none)
```

Comment:
i believe there is a `get_dagster_event()` with the coerced return type (throws if None)

Reply:
ah good to know